src/engine/javascript_core/engine_context.cc

```diff
@@ -29,6 +29,10 @@ auto Context::make_undefined() const -> Value {
           JSValueMakeUndefined(this->internal->context)};
 }
 
+auto Context::make_null() const -> Value {
+  return {this->internal->context, JSValueMakeNull(this->internal->context)};
```

Comment:
We can change it to this, and move it to `src/engine/common`, as its no longer JavaScriptCore specific

```suggestion
  return from(nullptr);
```